ipaserver/plugins/fasagreement.py

```diff
@@ -0,0 +1,270 @@
+#
+# FreeIPA plugin for Fedora Account System
+# See COPYING for license
+#
+"""User agreement for Fedora Account System
+
+Member users are stored in "memberUser" attribute while related groups are
+stored in "member" attribute. FreeIPA does not have a "memberGroup" attribute.
+"""
+from ipalib import Bool, Str
+from ipalib import errors
+from ipalib import output
+from ipalib.plugable import Registry
+from ipaserver.plugins.baseldap import (
+    LDAPObject,
+    LDAPSearch,
+    LDAPCreate,
+    LDAPDelete,
+    LDAPUpdate,
+    LDAPQuery,
+    LDAPRetrieve,
+    LDAPAddMember,
+    LDAPRemoveMember,
+    pkey_to_value,
+)
+from ipalib import _, ngettext
+from ipapython.dn import DN
+from ipaserver.plugins.internal import i18n_messages
+
+__doc__ = _(
+    """
```

Comment:
We don't need anything here, do we?

Reply:
I opened https://github.com/fedora-infra/freeipa-fas/issues/115